README.md

```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
 
 ### Step 1. Power up C3X, connect to Wi-Fi, and install `commaai/master-ci`
 
-1-1. Start with the official installation guide: https://comma.ai/setup/comma-3x
+1-1. Install the hardware by following the official Installation Guide: https://comma.ai/setup/comma-3x
```

Comment:
I wonder if it's worth linking to installation videos for Toyotas in general.

Reply:
Good idea

Reply:
The most comprehensive one is this but it talks about comma power and ethernet. May need a new one. https://www.youtube.com/watch?v=_VWCt4UMXUY

Reply:
Same, maybe on a later PR?